src/bumper.js

```diff
@@ -515,6 +517,8 @@ class Bumper extends BasePlugin implements IMiddlewareProvider, IAdsControllerPr
         this._bumperVideoElement.src = this.config.url;
         this._bumperVideoElement.setAttribute('playsinline', '');
       }
+    } else if (this._bumperState === BumperState.LOADED) {
+      this.dispatchEvent(EventType.AD_LOADED, {ad: this._getAd()});
```

Comment:
so you will get this line multiple times when only postroll exists no?

Reply:
מot even once, according to what I checked

Reply:
not even once, according to what I checked